packages/browser/src/posthog-core.ts

```diff
@@ -2417,8 +2434,23 @@ export class PostHog {
      * @public
      */
     opt_out_capturing(): void {
```

Comment:
Where are we doing the opposite of this - removing cookieless mode when someone who was opted out once and then opts in?

Reply:
You're right, at this point we should call reset() and *then* opt them in. I'll add this as a follow-up, as the first customer who wants to use this is going to use "always" which is unaffected.

Reply:
There's tests for this now